sdcm/nemesis.py

```diff
@@ -6382,6 +6389,9 @@ class RandomInterruptionNetworkMonkey(Nemesis):
     run_with_gemini = False
     kubernetes = True
 
+    additional_configs = ["configurations/network_config/two_interfaces.yaml"]
+    additional_params = {"ip_ssh_connections": "public"}
```

Comment:
The `sdcm/utils/nemesis_jobs_configs.py` file had a comment describing the perspective of this param.
Not it is missing. 
Need to port comment too.